installer/registry.go

```diff
@@ -114,11 +114,15 @@ func GetSupportedRegistry() registry {
 		// Ubuntu
 
 		// BYOH Bundle Repository. Associate bundle with installer
-		linuxDistro := "Ubuntu_20.04.1_x86-64"
-		reg.AddBundleInstaller(linuxDistro, "v1.24.*")
-		reg.AddBundleInstaller(linuxDistro, "v1.25.*")
-		reg.AddBundleInstaller(linuxDistro, "v1.26.*")
+		linuxDistro20_04 := "Ubuntu_20.04.1_x86-64"
+		linuxDistro22_04 := "Ubuntu_22.04_x86-64"
 
+		reg.AddBundleInstaller(linuxDistro20_04, "v1.24.*")
```

Comment:
1.24 to 1.26 and then 1.31 - nothing in between? (1.27 to 1.30)? 
We have to remember to edit this go code when we add support for 1.32 - and further versions in future? 
Can we read this from a json file so that the code doesn't have to change and recompilation is not required unless there's a real change in go code?

Reply:
Agree with Trilok. We also don't need all these older versions. We just need 1.31+

Reply:
Yea, sure!!....Removed the older versions and maintaining only the new versions